packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,84 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  --terra-consumer-color: #64696c;
```

Comment:
i'm not sure how I feel about this abstraction. It feels very difficult to follow and understand. I think I'd prefer just passing in the actual colors to a SASS function to the things we are overriding.